src/algebra/category/Module/simple.lean

```diff
@@ -0,0 +1,46 @@
+/-
+Copyright (c) 2022 Pierre-Alexandre Bazin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Pierre-Alexandre Bazin
+-/
+import category_theory.simple
+import algebra.category.Module.abelian
+import ring_theory.simple_module
+
+/-!
+# Simple objects in the category of `R`-modules
+
+We prove simple modules are simple objects in the category of `R`-modules.
+TODO : prove that reciprocally, a simple object in the category of `R`-modules is indeed
+a simple module.
+-/
+
+section category
+variables {R M : Type*} [ring R] [add_comm_group M] [module R M]
+open category_theory
+open Module
+
+/-- If the zero morphism is an epi then the codomain in trivial. -/
+def unique_of_epi_zero (N : Module R) [h : epi (0 : N ⟶ of R M)] : unique M :=
+unique_of_surjective_zero N ((Module.epi_iff_surjective _).mp h)
```

Comment:
```suggestion
/-- If the zero morphism is an epi then the codomain is trivial. -/
def unique_of_epi_zero (N : Module R) [h : epi (0 : N ⟶ of R M)] : unique M :=
unique_of_surjective_zero N ((Module.epi_iff_surjective _).mp h)
```

Reply:
I think this should move to `Module/basic.lean`.

Reply:
I moved  it to `Module/epi_mono.lean`, as that's where `epi_iff_surjective` is defined